src/tests/projects.test.js

```
@@ -254,4 +254,243 @@ describe('DELETE /projects/:id', () => {
+    const response = await request(app).get(`/projects/${projectId}`).send();
+    expect(response.body.message).toBe('Project found');
+    expect(response.statusCode).toBe(200);
+    expect(response.body.error).toBe(false);
+  });
+  test('It should get an array of one populate', async () => {
+    const response = await request(app).get(`/projects/${projectId}`).send();
+    expect(response.body.data.employees[0].employeeId._id).toBe('62840603549ef329a075ef63');
+    expect(response.body.data.employees[0].employeeId.firstName).toBe('Luciano');
+    expect(response.body.data.employees[0].employeeId.lastName).toBe('Alarcon');
+  });
+  test('It should return a status 404', async () => {
+    const response = await request(app).get('/projects/628a59c9f67d451615818847').send();
+    expect(response.body.message).toBe('Project not found');
+    expect(response.statusCode).toBe(404);
+    expect(response.body.error).toBe(true);
+  });
+  test('It should return a status 500', async () => {
+    const response = await request(app).get('/projects/6').send();
+    expect(response.body.message.message).toBe('Cast to ObjectId failed for value "6" (type string) at path "_id" for model "Project"');
+    expect(response.statusCode).toBe(500);
+    expect(response.body.error).toBe(true);
+  });
+});
+describe('POST /', () => {
+  test('It should create the project', async () => {
+    const response = await request(app).post('/projects/').send({
+      name: 'Radium Rocket',
+      status: 'active',
+      description: 'BaSP-TG-26: radium',
+      employees: [
+        {
+          rate: 1,
+          role: 'QA',
+          employeeId: '60d4a32f257e066e9495ce12',
+        },
+      ],
+      startDate: '2002-12-09T00:00:00.000+00:00',
+    });
+    expect(response.body.message).toBe('Project was created.');
+    expect(response.statusCode).toBe(201);
+    expect(response.body.error).toBe(false);
+  });
+  test('It should was error about the keys (name => nombre)', async () => {
```

`It should return an error about the keys (name => nombre)`